### PR TITLE
Add ISO Foreword

### PR DESCRIPTION
**Pull request by bact**
- Text from Bob's Word doc 2024-07-22
- Add once ISO JTC-1 approves and we are sending final spec to ISO
  - It is possible to be merge this first, without include this file to the chapter list in mkdocs.yml (currently the mkdocs.yml doesn't include this)
  - Once the final spec was sent, we can include the file to mkdocs.yml

This PR will supersede #974

### CODE REVIEW

**Review by bobmartin3000 — APPROVED** [review]
LGTM

File: docs/iso-foreword.md
Comment (kestewart) [inline docs/iso-foreword.md:38]: Should we be mentioning both organizations here?

Comment (bact) [reply docs/iso-foreword.md:38]: @kestewart another one is The Linux Foundation?

Comment (swinslow) [reply docs/iso-foreword.md:38]: I would suggest using the following:

> This document was prepared by The Linux Foundation under the SPDX Working Group (as SPDX® Specification v3.0.1) and drafted in accordance with its editorial rules. Its preparation and publication has been made in coordination with related efforts with the Object Management Group (OMG).

@bobmartin3000 @kestewart Are you comfortable with this phrasing?

**Comment by swinslow** [comment]
I just added a comment with proposed edits to the language that @kestewart flagged.

I'll ask also (but will defer to the tech team as I'm definitely not close to the details here!) -- it sounds like this language wouldn't be incorporated unless and until it's actually approved by ISO / JTC. Is that correct? Just making sure we aren't publishing something saying "this is approved by ISO" if it isn't actually yet.  :)

**Comment by bobmartin3000** [comment]
LGTM - Bob

From: Steve Winslow ***@***.***>
Date: Friday, August 9, 2024 at 3:55 PM
To: spdx/spdx-spec ***@***.***>
Cc: Robert A Martin ***@***.***>, Mention ***@***.***>
Subject: [EXT] Re: [spdx/spdx-spec] Add ISO Foreword (PR #1001)
@swinslow commented on this pull request. In docs/iso-foreword. md: > +Introduction and/or on the ISO list of patent declarations received +(see [www. iso. org/patents](https: //www. iso. org/patents)) or the IEC list of +patent declarations received


@swinslow commented on this pull request.

________________________________

In docs/iso-foreword.md<https://github.com/spdx/spdx-spec/pull/1001#discussion_r1712073965>:

> +Introduction and/or on the ISO list of patent declarations received

+(see [www.iso.org/patents](https://www.iso.org/patents)) or the IEC list of

+patent declarations received (see [patents.iec.ch](https://patents.iec.ch)).

+

+Any trade name used in this document is information given for the convenience

+of users and does not constitute an endorsement.

+

+For an explanation of the voluntary nature of standards, the meaning of ISO

+specific terms and expressions related to conformity assessment, as well as

+information about ISO's adherence to the World Trade Organization (WTO)

+principles in the Technical Barriers to Trade (TBT), see

+[www.iso.org/iso/foreword.html](https://www.iso.org/iso/foreword.html).

+In the IEC, see

+[www.iec.ch/understanding-standards](https://www.iec.ch/understanding-standards).

+

+This document was prepared by the Object Management Group (OMG) (as SPDX®

I would suggest using the following:

This document was prepared by The Linux Foundation under the SPDX Working Group (as SPDX® Specification v3.0.1) and drafted in accordance with its editorial rules. Its preparation and publication has been made in coordination with related efforts with the Object Management Group (OMG).

@bobmartin3000<https://github.com/bobmartin3000> @kestewart<https://github.com/kestewart> Are you comfortable with this phrasing?

**Comment by bobmartin3000** [comment]
That is the plan.
This would only appear in the document going to ISO.
Bob

From: Steve Winslow ***@***.***>
Date: Friday, August 9, 2024 at 3:57 PM
To: spdx/spdx-spec ***@***.***>
Cc: Robert A Martin ***@***.***>, Mention ***@***.***>
Subject: [EXT] Re: [spdx/spdx-spec] Add ISO Foreword (PR #1001)
I just added a comment with proposed edits to the language that @kestewart flagged. I'll ask also (but will defer to the tech team as I'm definitely not close to the details here!) -- it sounds like this language wouldn't be incorporated unless


I just added a comment with proposed edits to the language that @kestewart<https://github.com/kestewart> flagged.

I'll ask also (but will defer to the tech team as I'm definitely not close to the details here!) -- it sounds like this language wouldn't be incorporated unless and until it's actually approved by ISO / JTC. Is that correct? Just making sure we aren't publishing something saying "this is approved by ISO" if it isn't actually yet. :)

**Review by swinslow — APPROVED** [review]

**Comment by bact** [comment]
:( we got a conflicts again, can't rebased

File: docs/iso-foreword.md
Comment (bact) [reply docs/iso-foreword.md]: @swinslow I have added your suggestion to the source, with additional "and its Contributors" which I take from the copyright line from #1010

**Review by zvr — APPROVED** [review]

**Review by swinslow — APPROVED** [review]
I'm good with this, thank you @bact!

**Review by kestewart — APPROVED** [review]

**Pull request merged**
